docs/harfbuzz-sections.txt

```diff
@@ -569,6 +569,12 @@ hb_ot_math_get_min_connector_overlap
 hb_ot_math_get_glyph_assembly
 </SECTION>
 
+<SECTION>
+<FILE>hb-ot-metrics</FILE>
+hb_ot_metrics_t
+hb_ot_metrics_get
```

Comment:
Maybe `hb_ot_metrics_get_by_tag` instead?

Reply:
Hehe, what is left from my side?

Reply:
> Hehe, what is left from my side?

MVAR table implementation?

Reply:
I also left a few other suggestions / questions re using hmtx/vmtx accelerators instead of accessing hhea/vhea for ascent/descent/linegap...

Reply:
> MVAR table implementation?

Some similar code?

Reply:
Did you check the table?!  It's just a VarStore.  HVAR does the same...

Reply:
I did but not sure about the API use still, have a look.